kfp_notebook/pipeline/_notebook_op.py

```diff
@@ -87,33 +89,37 @@ def __init__(self,
                 NOTE: Images being pulled must have python3 available on PATH and cURL utility
             """
 
+            argument_list.append('mkdir -p ./%s && cd ./%s && '
+                                 'curl -H "Cache-Control: no-cache" -L %s --output bootstrapper.py && '
+                                 'curl -H "Cache-Control: no-cache" -L %s --output requirements-elyra.txt && '
+                                 'python -m pip install packaging &&'
+                                 'python -m pip freeze > requirements-current.txt &&'
+                                 'python bootstrapper.py '
+                                 ' --cos-endpoint %s '
+                                 ' --cos-bucket %s '
+                                 ' --cos-directory "%s" '
+                                 ' --cos-dependencies-archive "%s" '
+                                 ' --notebook "%s" ' % (
+                                     self.container_work_dir,
+                                     self.container_work_dir,
+                                     self.bootstrap_script_url,
+                                     self.requirements_url,
+                                     self.cos_endpoint,
+                                     self.cos_bucket,
+                                     self.cos_directory,
+                                     self.cos_dependencies_archive,
+                                     self.notebook
+                                     )
+                                 )
+
+            if self.pipeline_inputs:
+                argument_list.append(['--pipeline_inputs', self.pipeline_inputs])
```

Comment:
Should this (and `--pipeline_outputs` below) still be `--inputs` (and `--outputs`) - or we need to change the arg handler in bootstrapper.py